docusaurus.config.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
       items: [
         {
           to: 'docs/',
-          label: 'Quick Start',
+          label: 'Docs',
```

Comment:
The reason is that after changing to 'docsVersionDropdown,' the original versioned item name is no longer named 'Docs'. After referencing this page 'https://docusaurus.io/docs', I think changing 'Quick Start' to 'Docs' is better.'